features/tweet.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Tweet
+
+  Scenario: Tweet Creation
```

Comment:
Great! Read over his styleguide and make some formatting changes - super helpful. 

The way you suggested actually super simplified my tweet_steps and allowed them to be more generalized.